README.md

```diff
@@ -17,12 +17,18 @@ with privileged access to the machine that handles the data) has unrestricted
 access to the machine hardware resources, and can leverage that to bypass any
 security mechanism on the machine itself and extract secret keys and data.
 
-As part of Project Oak, data are end-to-end encrypted between _enclaves_, which
-are isolated computation compartments that can be created on-demand, and provide
-strong confidentiality, integrity, and attestation capabilities via a
-combination of hardware and software functionality. Enclaves protect data and
-code even from the operating system kernel and privileged software, and are
-intended to protect from most hardware attacks.
+[Oak Functions](/oak_functions) is a Trusted Runtime server binary that exposes
```

Comment:
Should Trusted Runtime also be defined as a term?

Reply:
Removed from here for now, but I agree we should think about that. Probably more related to the Transparent Release work anyways, so let's do another PR for that.